Workbooks/Workloads/Onboard MSSQL Insights/Onboard MSSQL Insights.workbook

```diff
@@ -214,23 +221,37 @@
                       {
                         "id": "0fcd1a59-1ca7-4e29-a0c7-6eb5b032ef6a",
                         "version": "KqlParameterItem/1.0",
-                        "name": "SqlConnectionString",
-                        "label": "Sql connection strings",
+                        "name": "TelegrafConfig",
+                        "label": "MSSQL connection strings",
                         "type": 1,
                         "isRequired": true,
-                        "value": "",
+                        "value": "[[inputs.sqlserver]]\r\n  servers = [\r\n   'Server=mysqlserver.database.windows.net;Port=1433;database=mydb;User Id=<user>;Password=<pw>;app name=telegraf;log=1;',\r\n  ]\r\n  query_version = 2\r\n  exclude_query = [ 'Schedulers' , 'SqlRequests']\r\n\r\n  [inputs.sqlserver.tags]\r\n    insight = \"mssql\"",

```

Comment:
Maybe we should change the sample server name so that it is not confused with MySQL. I didn’t think of this when I first came up wit the example.

Reply:
Can you change tag named ‘insight' to ‘in’ and add a tag for version called ‘ver’ in this and other configs. Let’s try to keep our metadata footprint as small as possible.

Reply:
Sure, in the next iteration I will change this and will also add the version tag and will also change the server name.